install/common/download-tessdata.py

```diff
@@ -0,0 +1,80 @@
+import hashlib
+import io
+import json
+import pathlib
+import re
+import subprocess
+import sys
+import tarfile
+import urllib.request
+
+TESSDATA_RELEASES_URL = (
+    "https://api.github.com/repos/tesseract-ocr/tessdata_fast/releases/latest"
+)
+TESSDATA_ARCHIVE_URL = "https://github.com/tesseract-ocr/tessdata_fast/archive/{tessdata_version}/tessdata_fast-{tessdata_version}.tar.gz"
+TESSDATA_CHECKSUM = "d0e3bb6f3b4e75748680524a1d116f2bfb145618f8ceed55b279d15098a530f9"
+
+
+def git_root():
```

Comment:
You're right, I always wanted to do this as well. My only reservation was a stylistic one. Because our dev scripts are regular Python scripts, we can't create a `util.py` module and peform a relative import:

```python
from . import util
```

since we'd get the following error:

```
ImportError: attempted relative import with no known parent package
```

Instead, what we can do is name the module in a way that would not class with other top-level modules, and perform an absolute import. Let me know if 6a5b6e4249e7818ca67ffde47578469d05e26382 works for you.

Reply:
Oh, actually, we have a problem. There's no common module for scripts in `install/` and `dev_scripts`. So, there's bound to be some duplication there. I've reverted this change for now, since it's breaking our tests, but let me know if you have an elegant solution to this.

Reply:
The main reason why we cannot use relative imports when using scripts is that it would require adding `/dev_scripts/__init__.py` and then using `python -m dev_scripts.env`, or an entrypoint doing this for us.

One solution to this problem would be to have the scripts being packaged somehow by dangerzone (in `dangerzone.scripts` or `dangerzone.tools`, for instance), and packaged by pyproject.toml so we have ways to run the scripts.

I believe this ties to the other discussion we're having about being able to run the scripts without installing dependencies, but that's my preferred way I think.

Another solution would be to put these scripts in the same folder, somehow, and then use `python -m scripts.env` or `python -m scripts.build_image`, possibly putting these in a `Makefile` to ease the flow.

Also, it's possible to [mess with the python path](https://stackoverflow.com/a/49375740/147077)

Reply:
You're right, it ties with the other discussion we have about third-party packages in our developer scripts. Basically, we currently have scripts that are kind of standalone, and we can improve them to:
1. Import code from a common place.
2. Use third-party packages.
3. Maybe even check them with MyPy, as we do for our regular code.

In my opinion, it's best to create a separate issue (something like "Convert dev scripts into their own Python project"), so that we can scope it properly, discuss solutions, and prioritize its implementation. As is, most likely it doesn't have to block this PR.

What do you think?

Reply:
You're right! Closing this to not block the merge. Let's discuss this there: https://github.com/freedomofpress/dangerzone/issues/946